webapps/assembly/pom.xml

```
@@ -4,63 +4,25 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.camunda.bpm</groupId>
-    <artifactId>camunda-database-settings</artifactId>
+    <groupId>org.camunda.bpm.webapp</groupId>
+    <artifactId>camunda-webapp-root</artifactId>
     <version>7.19.0-SNAPSHOT</version>
-    <relativePath>../../database</relativePath>
+    <relativePath>../</relativePath>
   </parent>
 
   <artifactId>camunda-webapp</artifactId>
   <groupId>org.camunda.bpm.webapp</groupId>
```

```suggestion
  <groupId>org.camunda.bpm.webapp</groupId>
```
🙃 this is redundant.

---

Right, over here it's redundant 🙂